package.json

```diff
@@ -31,4 +30,4 @@
   "scripts": {
     "test": "mocha -t 500000"
   }
-}
+}
```

Comment:
Minor: I am not sure what is the general rule for line ending in JavaScript world?